### ISSUE TITLE
Escape key should cancel UserSettings

### ISSUE THREAD

**Issue by ara4n** [vector-im/riot-web#3267]

**Comment by t3chguy** [comment]
I presume it should also cancel RoomSettings, and therefore be an issue within matrix-react-sdk as thats where the components originate. Seems like a nicer fix than closing them from this end, or am I missing something?

**Comment by t3chguy** [comment]
This is in experimental right now
